azure-pipelines.yml

```diff
@@ -130,31 +130,81 @@ stages:
       displayName: Run ITs
       strategy:
         matrix:
-          Idea2018:
-            IDEA_VERSION: '2018.3'
-            JDKVersion: '1.11'
+          # remote-robot crashes on 2018.3
           Idea2019:
-            IDEA_VERSION: '2019.2'
-            JDKVersion: '1.11'
+            IDEA_VERSION: 'IC-2019.3'
+          Idea2020:
+            IDEA_VERSION: 'IC-2020.1.3'
```

Comment:
Can we add LATEST EAP?

Reply:
We could use `LATEST-EAP-SNAPSHOT` yes. I was a bit reluctant because I don't know if it would break our tests too often. Maybe best way is to try and see! I'll make the change